### DISCUSSION TITLE
Failed to initialize webgl

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#3310] [Q&A] [answered]
How can I allow the webdriver to initialize webgl?
![image](https://github.com/user-attachments/assets/660158c3-6276-42c5-a523-d9fd6c37a907)

**Comment by mdmintz** [comment, marked as answer]
Set `enable_3d_apis=True`.

Example: [SeleniumBase/examples/test_3d_apis.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_3d_apis.py)
